README.md

```diff
@@ -42,14 +42,21 @@ Join the Aim community on <a href="https://slack.aimstack.io">Slack</a>
 
 </div>
 
-## Getting started in three steps
+Aim is designed to... @gevord please add the intro here.
```

Comment:
@SGevorg input needed here

Reply:
With more resources and complex models more experiments are ran than ever.

Aim is an open-source comparison tool AI experiments. Use Aim to deeply inspect 1000s of hyperparameter-sensitive training runs at once.